test/resources/controls/aws_parallelcluster_install/lustre_spec.rb

```diff
@@ -0,0 +1,58 @@
+control 'lustre_client_installed' do
+  title "Verify that lustre client is installed"
+
+  if (os_properties.centos? && inspec.os.release.to_f >= 7.5) || os_properties.redhat?
+    describe package('kmod-lustre-client') do
+      it { should be_installed }
+    end
+
+    describe package('lustre-client') do
+      it { should be_installed }
+    end
+
+    if (os_properties.centos? && inspec.os.release.to_f >= 7.7) || os_properties.redhat?
+      describe yum.repo('aws-fsx') do
+        it { should exist }
+        it { should be_enabled }
+        its('baseurl') { should include 'fsx-lustre-client-repo.s3.amazonaws.com' }
+      end
+    end
```

Comment:
Nitpick. Not sure we need this check. We need packages to be installed, not the repo to be configured. Why should it matter?

Reply:
A thing that came to my mind is to ensure we download the packages from the right "authorative" repo that is specified in the documentation

https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html

anyway if we don't want to do this check we may remove the tests yes, won't do it in this PR